### PR TITLE
Disable format button without Prettier applied

### LINKED ISSUE #3278: Disable format button without prettier module
The format button should be disabled if the prettier module is not applied

### PR DESCRIPTION
**Pull request by wmarques**
Fix #3278

### CODE REVIEW

File: src/test/javascript/spec/module/primary/landscape/LandscapeComponent.spec.ts [hunk 1]
```diff
@@ -370,7 +370,8 @@ describe('Landscape', () => {
       const [appliedModules] = modules.applyAll.lastCall.args as ModulesToApply[];
       expect(appliedModules.modules.map(slug => slug.get())).toEqual(['vue']);
       expect(wrapper.find(wrappedElement('modules-apply-new-button')).attributes('disabled')).toBeDefined();
-
+      const component: any = wrapper.vm;
+      expect(component.isApplied('vue')).toBeTruthy();
```
Comment (hdurix) [inline src/test/javascript/spec/module/primary/landscape/LandscapeComponent.spec.ts:374]: maybe another assert with a not applied one?

File: src/main/webapp/app/module/primary/landscape/Landscape.html [hunk 2]
```diff
@@ -33,7 +33,12 @@
       @propertyDeleted="deleteProperty"
     />
 
-    <ProjectActionsVue :folderPath="folderPath" @operationStarted="operationStarted()" @operationEnded="operationEnded()">
+    <ProjectActionsVue
+      :folderPath="folderPath"
+      :isPrettierButtonEnabled="isApplied('prettier')"
```
Comment (hdurix) [inline src/main/webapp/app/module/primary/landscape/Landscape.html:38]: don't forget the patches component (could be `true` for now)

**Comment by wmarques** [comment]
The tests are failing because the "Format" Button are disabled by default, as "Prettier" is not applied, I see two ways:
* Mock "isApplied" function to always return true
* Click on "Prettier" in each failing test to enable the format button

Which one is better ? 
cc @hdurix

**Comment by hdurix** [comment]
@wmarques I tried to fix it with a PR on this PR: https://github.com/wmarques/jhipster-lite/pull/1

is it what you intended?

**Comment by wmarques** [comment]
Yes indeed ! Thanks ! I'll fix merge conflict tomorrow and hope it will be good

**Review by hdurix — APPROVED** [review]

**Pull request merged**